embed/config.go

```diff
@@ -36,7 +38,6 @@ import (
 	"go.etcd.io/etcd/pkg/transport"
 	"go.etcd.io/etcd/pkg/types"
 
-	"github.com/ghodss/yaml"
```

Comment:
replace this line with sigs.k8s.io/yaml instead of creating a new import block at line 30

Reply:
placed this at the bottom instead of simply replacing the existing ghodss dependency as the 3rd party imports should be sorted alphabetically.